src/condor_scripts/condor_remote_cluster

```diff
@@ -860,12 +860,12 @@ if [ -z "${release_url}" ]; then
     fi
 
     release_url="${url_base}${series}/${version}/release/condor-${version}-${platform}-stripped.tar.gz"
-    update_url="${url_base}${series}/${version}/update/condor-${version}-${platform}-stripped.tar.gz"
+    beta_url="${url_base}${series}/${version}/beta/condor-${version}-${platform}-stripped.tar.gz"
```

Comment:
The beta release is an RC and really belongs in the immediately following if block with the alpha release.